package pl.codewise.test.utils;

public class MethodCallException extends RuntimeException {
    public MethodCallException(String message, Throwable cause) {
        super(message, cause);
    }
}
